### DISCUSSION TITLE
[Github Pages] it would be nice we could run PHP code here in Github Pages

### DISCUSSION THREAD

**Discussion by sebiboga** [community/community#6924] [Other Feature Feedback, Questions, & Ideas]
For now we understand Github Pages shows only static content. 
Would it be possible to enable also PHP right there in Github Pages?

Thank you

**Comment by jfullstackdev** [comment]
that's always possible, the question is whether GitHub is willing to implement that,

just to give you a background, GitHub Pages was created for the need of a dedicated documentation page, which would not require server-side scripting at all, just simply a static site
